index.js

```diff
@@ -533,29 +529,26 @@ export class Placement {
  * @returns {Promise<[Placement]>} Resolves to a list of Placement instances
  */
 export function load_placements(force_load = false) {
-  // Find all elements matching required data binding attribute. We don't yet
-  // support multiple placements on the ad-server. For now, this could result in
-  // competing ad placements.
+  // Find all elements matching required data binding attribute.
   const node_list = document.querySelectorAll("[" + ATTR_PREFIX + "publisher]");
   let elements = Array.prototype.slice.call(node_list);
 
-  // Create main promise. Iterator `all()` Promise wil surround array of found
+  // Create main promise. Iterator `all()` Promise will surround array of found
   // elements. If any of these elements have issues, this main promise will
   // reject.
   if (elements.length === 0) {
     throw new Error("No ad placements found.");
-  } else if (!SUPPORTS_MULTIPLE_PLACEMENTS && elements.length > 1) {
-    console.error(
-      "Multiple ad placements are not supported, only using the first ad placement."
-    );
-    elements = elements.slice(0, 1);
   }
 
   return Promise.all(
```

Comment:
Does this change the fallback logic that we document for using a fallback ad? https://ethical-ad-client.readthedocs.io/en/latest/#showing-content-when-there-isn-t-an-ad

I don't know JS well enough to know, but I think we probably need to improve that API in some fashion to take a placement name or index?

Reply:
I verified that this change does not affect that logic.